support/objc-support.m

```diff
@@ -49,3 +49,15 @@ NSComparisonResult mono_embeddinator_compare_to (MonoEmbedObject *object, MonoMe
 	void* __unbox = mono_object_unbox (__result);
 	return (NSComparisonResult) *((int*)__unbox);
 }
+
+MonoObject* mono_embeddinator_get_object (id native, bool assertOnFailure)
+{
+	if (![native respondsToSelector:@selector (xamarinGetGCHandle)]) {
```

Comment:
Should this assert if `native=nil` and `assertOnFailure=true` (I believe it currently does, but IMHO it might make more sense to return `nil` in that case)?